crd/crdtype/controller_type.go

```diff
@@ -0,0 +1,39 @@
+package crdtype
+
+import (
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"github.com/rancher/longhorn-manager/crd/tools/crdcopy"
+	"github.com/rancher/longhorn-manager/types"
+)
+
+const (
+	ControllerPlural      string = "crdcontrollers"
+	ControllerFullName    string = ControllerPlural + "." + CRDGroup
+	ControllerShortname	   string = "cc"
+)
+
+type Crdcontroller struct {
```

Comment:
Don't need to add `Crd` prefix since it's in the different package, there will be no name conflict. Also, It's title case and CRD should be all cap, so it can be `CRDController` instead of `Crdcontroller`.